kafka-ansible/requirements.txt

```diff
@@ -1,3 +1,4 @@
 ansible>=2.4.2
```

Comment:
resrict ansible version
refer to https://github.com/pingcap/tidb-ansible/blob/master/requirements.txt#L1
This file containes python packages requirement, see https://pip.pypa.io/en/stable/user_guide/#id12